process/transaction/shardProcess.go

```diff
@@ -262,6 +282,11 @@ func (txProc *txProcessor) processMoveBalance(
 		return err
 	}
 
+	err = txProc.checkIfValidTxToMetaChain(tx, acntSrc, adrDst)
```

Comment:
I would rename this method to checkTxValidity as it is called with any kind of tx no matter if the destination is meta or not.

Reply:
there is already a checkTxValidity method - this is checking only in case of metachain calls.

Reply:
Ok. Anyway I think that a better name could be found like: checkTxValidityIfDestIsMetachain or something like this